pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+markers =
+    smoke: mark a test as a smoke test
+    flaky: mark a test as a flaky test for rerun
```

Comment:
We probably want to centralize all configs into `pyproject.toml`

Reply:
Yup that would be much cleaner!